BorrowToOwn.API/Controllers/ProductController.cs

```diff
@@ -31,11 +31,12 @@ public ProductController(IProductService productService,IHttpClientFactory httpC
          DisableRequestSizeLimit,
          RequestFormLimits(ValueLengthLimit = int.MaxValue, MultipartBodyLengthLimit = long.MaxValue)]
         [Consumes("multipart/form-data")]
-        public async Task<IActionResult> Post([FromForm] ProductRequestObject PaymentPlanRequestObject)
+        public async Task<IActionResult> Post([FromForm] ProductRequestObject productRequestObject)
```

Comment:
I think i already asked why FromForm and not FromBody...... just dropping it again if i didn't ask before

Reply:
ehm , its for form data because of the form files that will be sent across , the endpoint takes files